bin/bootstrap

```diff
@@ -27,7 +31,7 @@ if ! command -v npm >/dev/null 2>&1; then
 fi
 
 info 'Checking for `.env` environment variables file.'
-if [ ! -e ".env" ]; then
+if [ ! -e "$PROJECT_ROOT/.env" ]; then
```

Comment:
This shouldn't be necessary since the script is starting by changing to the PROJECT_ROOT directory.